.travis.yml

```diff
@@ -64,7 +64,7 @@ script:
   # complexity (--mccabe) and time (--profile, --durations) analysis by pytest
   # are too simplistic and buggy currently
   - python -m pytest --cov --pastebin=failed ;
-  - pprofile $(which py.test) ;
+  # - pprofile $(which py.test) ;
```

Comment:
I apologize for this. What would be the way of testing such a thing on travis w/o doing such commits?